### ISSUE TITLE
Keyboard.SendText doesn't work in x64 config

### ISSUE THREAD

**Issue by j3soon** [j3soon/LowLevelControls#1]
The following code works in `AnyCPU` and `x86`, but not in `x64` config.

```cs
using System.Threading;

namespace Test
{
    class Program
    {
        static void Main(string[] args)
        {
            Thread.Sleep(3000);
            Keyboard.SendText("ABCD");
        }
    }
}
```

**Comment by j3soon** [comment]
Related Links:
- https://stackoverflow.com/a/6833311/3917161
- https://social.msdn.microsoft.com/Forums/vstudio/en-US/cf429cdf-146d-48c9-a8a2-cec9fc4b298a/sendinput-wont-work-when-compiled-with-x64-option?forum=csharpgeneral

**Comment by j3soon** [comment]
Fixed via 5349e430b2e0c711c4e4060a42b12d86abf5be08
